ui/src/app/machines/views/MachineDetails/MachineNetwork/BondForm/BondFormFields/BondFormFields.tsx

```diff
@@ -78,6 +48,15 @@ const BondFormFields = ({ selected, systemId }: Props): JSX.Element | null => {
           component={Select}
           label="Link monitoring"
           name="linkMonitoring"
+          onChange={(evt: React.ChangeEvent<HTMLSelectElement>) => {
+            const { value } = evt.target;
+            // Manually set the value because we've overwritten the onChange.
```

Comment:
Would formik's `handleChange(e)` handler work here? I've used it a couple of times when I only want to extend the `onChange` rather than overwrite it entirely, but I can't remember if it's got some idiosyncrasies.

Reply:
I think it will, thanks!

Reply:
Yep that worked.